conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class DXCConan(ConanFile):
+    name = "dxc"
+    version = "1.6.2112"
+    description = "DirectX Shader Compiler"
+    license = "NCSA"
+    topics = ("hlsl", "dxc", "compiler", "shader", "spirv")
+    homepage = "https://github.com/microsoft/DirectXShaderCompiler"
+    url = "https://github.com/triadastudio/conan-dxc"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    no_copy_source = True
+
+    @property
+    def _source_commit_or_tag(self):
+        return "v1.6.2112"
```

Comment:
Why not just `return self.version`

Reply:
It is actually different from the version string (with `v` prefix)
and in later updates it will often be the commit ref.